pkg/controller/run/main_internal_test.go

```diff
@@ -136,3 +136,49 @@ func TestController_patchLine(t *testing.T) {
 		})
 	}
 }
+
+func TestController_getConfigPath(t *testing.T) {
```

Comment:
Should I add test for `readConfig` instead of this extracted `getConfigPath` function?